studio/test/store/hash-linking.test.js

```diff
@@ -0,0 +1,50 @@
+import { expect } from '@esm-bundle/chai';
+import Store, { linkStoreToHash, unlinkStoreFromHash } from '../../src/store';
```

Comment:
pleas use `store.js` extension, in unit tests WTR adds it automatically but such imports do not work in runtime.
e.g: 
![image](https://github.com/user-attachments/assets/96423353-6640-4f79-94f4-bcd078031b63)

Reply:
I just need the file extension, then I'm good, will approve.